Services/Normalizer/ConsignmentNormalizer.php

```diff
@@ -22,9 +22,9 @@ public function __construct(?array $data)
     public function normalize(): array
     {
         $data                 = $this->data;
-        $data['carrier']      = $data['carrier'] ?: PostNLConsignment::CARRIER_NAME;
+        $data['carrier']      = $data['carrier'] ?? PostNLConsignment::CARRIER_NAME;
         $data['deliveryType'] = $data['deliveryType'] ?? AbstractConsignment::DELIVERY_TYPE_STANDARD_NAME;
-        $data['package_type'] = $data['package_type'] ?? AbstractConsignment::PACKAGE_TYPE_PACKAGE;
+        $data['packageType']  = $data['package_type'] ?? AbstractConsignment::PACKAGE_TYPE_PACKAGE_NAME;
```

Comment:
Hier gaat denk ik iets mis als de check op `package_type` is en de waarde in `packageType` komt...